pypeit/scripts/chk_calibs.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script examines a set of files and indicates which do and
+which do not have sufficient calibs
+"""
+import argparse
+from pypeit import defs
+
+from IPython import embed
+
+def parser(options=None):
+    # TODO: Add argument that specifies the log file
+    parser = argparse.ArgumentParser(description="Script to setup a PypeIt run [v3]")
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-r', '--root', type=str, default=None,
+                       help='File path+root, e.g. /data/Kast/b ')
```

Comment:
Don't need a mutually exclusive group here.  Since it's always required, I would remove `group` and set:
```
parser.add_argument('root', type=str, default=None, help='File path+root, e.g. /data/Kast/b ')
```

And then the call would be, e.g.:
```
pypeit_chk_calibs /PypeIt-development-suite/RAW_DATA/not_alfosc/grism4/ALDc2 -s not_alfosc
```

Unless it's important to mimic the `pypeit_setup` call.

Reply:
done!